RELEASE_NOTES.md

```diff
@@ -72,7 +72,7 @@ environment is setup, please refer to [SETUP.md](./docs/SETUP.md).
 
 For provisioning Greengrass nucleus lite devices by claim certificates, please
 take a look at the fleet provisioning by claim setup guide
-[here](./docs/FLEET_PROVISIONING.md).
```

Comment:
Why are these sections in the release notes?? just remove the installing from source and contribution guidlines from the release notes.